src/plugins/googledrive/admin/src/pages/HomePage/index.js

```diff
@@ -0,0 +1,19 @@
+/*
+ *
+ * HomePage
```

Comment:
do we need this comment to explain that this is the HomePage? It seems clear to me because you did a great job of naming this folder and component on line 10. (code seems to be self documenting)